### PR TITLE
Fix export tutorial

### PR DESCRIPTION
**Pull request by angelayi**
Fixing issues from @lucylq (thanks!)

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

### CODE REVIEW

**Review by lucylq — APPROVED** [review]

**Comment by svekars** [comment]
I believe we need to enable here: https://github.com/pytorch/tutorials/blob/main/.jenkins/validate_tutorials_built.py#L52

File: intermediate_source/torch_export_tutorial.py [hunk 1]
```diff
@@ -280,25 +223,27 @@ def false_fn(x):
...
+print(exported_map_example)
+print(exported_map_example.module()(*inps))
+
+######################################################################
+# Other control flow ops include ``while_loop``, ``associative_scan``, and
+# ``scan``. For more documentation on each operator, please refer to 
+# `this page <https://github.com/pytorch/pytorch/tree/main/torch/_higher_order_ops>`__.
 
 ######################################################################
 # Constraints/Dynamic Shapes
```
Comment (angelayi) [inline intermediate_source/torch_export_tutorial.py:249]: This section will be updated by https://github.com/pytorch/tutorials/pull/3214/

File: intermediate_source/torch_export_tutorial.py [hunk 2]
```diff
@@ -570,30 +511,27 @@ def forward(self, x):
...
-#
-# If you have a custom operator implemented in C++, please refer to
-# `this document <https://docs.google.com/document/d/1_W62p8WJOQQUzPsJYa7s701JXt0qf2OfLub2sbkHOaU/edit#heading=h.ahugy69p2jmz>`__
-# to make it compatible with ``torch.export``.
+# Note that in the ``ExportedProgram``, the custom operator is included in the graph.
 
 ######################################################################
-# Decompositions
-# --------------
+# IR/Decompositions
```
Comment (angelayi) [inline intermediate_source/torch_export_tutorial.py:522]: cc @tugsbayasgalan -- I updated the docs for run_decompositions

**Review by svekars — APPROVED** [review]
Looks good. We'll merge into main when 2.6 binaries are pulled there.
